RossBlassingame/JPL-CUSeniorProjects
# 59 - Mars unlimited scope algorithm

What changes were made:
Implemented the unlimited scope algorithm. Changed most files/classes to implement the Coordinate class.

What are the relevant JIRA tickets:
59

How can these changes be tested(test #?):
Need to create tests. Manually look at map, calculate slopes, and assert the route is the same as the manually checked route?

Comments:
The slope checking methods don't seem to be working, and currently the algorithm doesn't check for map boundaries. @robertBallard could you take a look at the getSlope and canTraverse methods? Once those are fixed, unlimited scope algorithm should be (mostly) complete.
